### PR TITLE
Add hugetlb stats 

### PR DESCRIPTION
**Pull request by sulixu**
Adding /sys/fs/cgroup/hugetlb statistics as discussed in #2406

### CODE REVIEW

**Comment by sulixu** [comment]
@googlebot I signed it!

File: info/v1/container.go [hunk 1]
```diff
@@ -833,12 +844,12 @@ type ProcessStats struct {
...
-	Cpu       CpuStats     `json:"cpu,omitempty"`
-	DiskIo    DiskIoStats  `json:"diskio,omitempty"`
-	Memory    MemoryStats  `json:"memory,omitempty"`
-	Network   NetworkStats `json:"network,omitempty"`
-
+	Timestamp time.Time               `json:"timestamp"`
+	Cpu       CpuStats                `json:"cpu,omitempty"`
+	DiskIo    DiskIoStats             `json:"diskio,omitempty"`
+	Memory    MemoryStats             `json:"memory,omitempty"`
+	Hugetlb   map[string]HugetlbStats `json:"hugetlb"`
```
Comment (dashpole) [inline info/v1/container.go]: I believe we should tag this with omitempty.

**Comment by sulixu** [comment]
> Can you update cadvisor.go as well? This should probably not be enabled by default, and should at least be possible to disable.

**Comment by sulixu** [comment]
> Can you update cadvisor.go as well? This should probably not be enabled by default, and should at least be possible to disable.

I've make hugetlb disabled by default, to enable it need: 
--disable_metrics=tcp,udp,advtcp,sched,process

tested with: 
1. --disable_metrics=tcp,udp,advtcp,sched,process,hugetlb with no container_hugetlb_* stats
2.--disable_metrics=tcp,udp,advtcp,sched,process,  container_hugetlb_* stats showed up

**Comment by dashpole** [comment]
```
I0317 00:51:05.780] >> checking go formatting
I0317 00:51:06.572] The following files are not properly formatted:
I0317 00:51:06.572] container/factory.go
```

**Pull request merged**